public/pages/Snapshots/components/RestoreActivitiesPanel/RestoreActivitiesPanel.tsx

```diff
@@ -28,17 +30,32 @@ export const RestoreActivitiesPanel: React.FC<RestoreActivitiesPanelProps> = ({
   const [flyout, setFlyout] = useState(false);
 
   useEffect(() => {
-    context?.chrome.setBreadcrumbs([BREADCRUMBS.SNAPSHOT_MANAGEMENT, BREADCRUMBS.SNAPSHOTS, BREADCRUMBS.SNAPSHOT_RESTORE]);
-    getRestoreStatus();
-  }, []);
+    context!.chrome.setBreadcrumbs([BREADCRUMBS.SNAPSHOT_MANAGEMENT, BREADCRUMBS.SNAPSHOTS, BREADCRUMBS.SNAPSHOT_RESTORE]);
+
+    let getStatusInterval: ReturnType<typeof setInterval>;
+
+    if (stage.slice(0, 4) !== "Done") {
```

Comment:
What's the difference if write like `stage !== "Done"` ?

Reply:
The percentage is part of the 'stage' string, at least when this PR was made.  Just slicing away the percentage part of the string.